scripts/tests/chiptest/runner.py

```diff
@@ -124,11 +124,15 @@ def get(self):
         return self.queue.get()
 
 
+class SubprocessKind(enum.Enum):
```

Comment:
Let's synchronize with https://github.com/project-chip/connectedhomeip/pull/42321 where it's a StrEnum with explicit values. Explicit values are required for `--app-path` and `--tool-path` parsing to work:

```python
class SubprocessKind(StrEnum):
    APP = 'app'
    TOOL = 'tool'
```

Reply:
Fixed